scripts/test-tls13-no-unknown-groups.py

```diff
@@ -0,0 +1,407 @@
+# Author: Alicja Kario, (c) 2024
```

Comment:
2025?

Reply:
ah, true... though I've written most of it last year, so... meh :slightly_smiling_face: